core/src/main/java/com/flipkart/gjex/core/config/RotationManagementConfig.java

```diff
@@ -0,0 +1,24 @@
+package com.flipkart.gjex.core.config;
```

Comment:
I do not see any usage of this ResourceConfig. Can we please remove it ?

Reply:
removed